sdk/loadtestservice/Azure.Developer.LoadTesting/samples/Sample01_CreateOrUpdateTest.md

```diff
@@ -0,0 +1,65 @@
+# Create or Update Load Test
```

Comment:
please follow our guide to write the samples https://github.com/Azure/azure-sdk-for-net/blob/main/doc/DataPlaneCodeGeneration/Create_Samples_Guidance.md. Thanks

And this is our sample RP https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/template/Azure.Template FYI

Reply:
Resolved